etc/PACKAGE_FILES_MAPPING

```diff
@@ -0,0 +1,3 @@
+export DAQ_LIB=/usr/lib/daq
```

Comment:
Maybe put this in debian/FILES_MAPPING?    I'd prefer not to have "FILES_MAPPING" and "PACKAGE_FILES_MAPPING" in the same place without qualifying FILES_MAPPING (e.g. SETUP_FILES_MAPPING rather than FILES_MAPPING).  Either way (either different directories or two unique prefixes) I think would work.

(Actually, if they're in the same place I'd recommend a suffix not a prefix so they're sorted together, e.g. FILES_MAPPING_SETUP and FILES_MAPPING_DEBIAN or something...)